src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -112,117 +112,121 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     ngraph::pass::Manager manager(get_pass_config());
     manager.set_per_pass_validation(false);
 
-    manager.register_pass<ov::pass::DisableDecompressionConvertConstantFolding>();
+    REGISTER_PASS(manager, ov::pass, DisableDecompressionConvertConstantFolding, )
 
     // Disable low_precision_enabled as all plugins handle low-precision sub-graph manually
     // before CommonOptimization pipeline execution
-    manager.register_pass<ngraph::pass::MOCTransformations>(true, false);
+    REGISTER_PASS(manager, ngraph::pass, MOCTransformations, _run_on_function, true, false)
 
     // Enabling conversion of FP16 IR to legacy representation, each plugin have to disable it
     // after support for FP16 IR is implemented
-    manager.register_pass<ov::pass::ConvertCompressedOnlyToLegacy>();
-
-    manager.register_pass<ov::pass::MarkPrecisionSensitiveDivides>();
-
-    // TODO: move to KMB
-    manager.register_pass<ngraph::pass::WeightsDequantizeToFakeQuantize>();
-
-    auto common_fusions = manager.register_pass<ngraph::pass::GraphRewrite>();
-    common_fusions->add_matcher<ngraph::pass::SpaceToBatchFusion>();
-    common_fusions->add_matcher<ngraph::pass::BatchToSpaceFusion>();
-    common_fusions->add_matcher<ngraph::pass::InterpolateSequenceFusion>();
-    common_fusions->add_matcher<ngraph::pass::SkipGatherBeforeTransposeAndReshape>();
-    common_fusions->add_matcher<ngraph::pass::ReduceMerge>();
-    common_fusions->set_name("ngraph::pass::CommonFusions");
+    REGISTER_PASS(manager, ov::pass, ConvertCompressedOnlyToLegacy, _run_on_model)
+    REGISTER_PASS(manager, ov::pass, MarkPrecisionSensitiveDivides, _run_on_model)
+    REGISTER_PASS(manager, ngraph::pass, WeightsDequantizeToFakeQuantize, )
```

Comment:
Just as idea:
```suggestion
    {
        using namespace ov::pass;
        using namespace ngraph::pass;
        REGISTER_PASS(manager, ConvertCompressedOnlyToLegacy)
        REGISTER_PASS(manager, MarkPrecisionSensitiveDivides)
        REGISTER_PASS(manager,  WeightsDequantizeToFakeQuantize)
    }
```

Reply:
If we add namespace here, is it possible to have a naming conflict. 
Honestly, I am not sure either.

Reply:
You use `using` only for localised places.